api/market_maker/market_maker.py

```diff
@@ -20,38 +25,47 @@
     pip3 install requests websockets
 """
 
-import asyncio
-import base64
-import decimal
-import hmac
-import json
 import os
-import random
 import sys
+import json
+import base64
+from random import randint, choice
+import random
+import hmac
 import time
-from random import randint
 from urllib.parse import urljoin
+import asyncio
+import decimal
+import traceback
+import argparse
 
-import requests
 import websockets
-
+import requests
 
 try:
     PARADIGM_ACCESS_KEY = sys.argv[1]
     PARADIGM_SECRET_KEY = sys.argv[2]
+    PARADIGM_ACCOUNT_NAME_DBT = sys.argv[3]
+    PARADIGM_ACCOUNT_NAME_BIT = sys.argv[4]
+    PARADIGM_ACCOUNT_NAME_CME = sys.argv[5]
+    PARADIGM_DESK_NAME = sys.argv[6]
 except IndexError:
     PARADIGM_ACCESS_KEY = None
     PARADIGM_SECRET_KEY = None
+    PARADIGM_ACCOUNT_NAME_DBT = None
+    PARADIGM_ACCOUNT_NAME_BIT = None
+    PARADIGM_ACCOUNT_NAME_CME = None
+    PARADIGM_DESK_NAME = None
 
-
+# Main Function
 async def main(access_key, secret_key, paradigm_account_information,
                paradigm_ws_url, dbt_http_host, bit_http_host,
                paradigm_http_host, min_tick_size):
     """
-    Connect to the websocket and respond to RFQs with quotes as they come in.
+    Primary async function.
     Initially subscribe to all notification channels.
     """
-    async with websockets.connect(f'{paradigm_ws_url}?api-key={access_key}') as websocket:
+    async with websockets.connect(f"{paradigm_ws_url}?api-key={access_key}") as websocket:
```

Comment:
Single quotes should be used here. General rule is to only use double quotes when dealing with a string that's being exposed to the user (an error message for example).
Am seeing it's the same comment for most changes involving single/double quotes. Please go through and revert the changes throughout.